### DISCUSSION TITLE
CBR to PDF

### DISCUSSION THREAD

**Discussion by darkrats** [sumatrapdfreader/sumatrapdf#3869] [General]
Earlier versions had the option under "save as" to save a CBR file as a PDF file. It appears to be missing. Was it removed?

**Comment by GitHubRulesOK** [comment]
yes most reliable version was 3.1.2, most uptodate version for export was 3.4.# but there were some modern image formats not possible to convert into PDF such as webp heic and a few others. It also means some txt exports are also not possible with newer versions.

**Reply by darkrats** [reply]
That's too bad. I guess I'll have to go back to 3.4.6.
